### ISSUE TITLE
SPIKE: Investigate a controlled testing environment

### ISSUE THREAD

**Issue by katevoss** [DataBiosphere/saturn-ui#677]
Investigate the feasibility of creating a controlled testing environment. This environment would have a set of users with different permission levels, data of different sorts, and other useful assets in order to test UI and functionality changes. This controlled environment would be reset on a specific cadence to prevent build-up of workspaces, billing projects, users, data, etc that has fallen into a bad state and/or is no longer relevant. 

The first step is to write a tech doc explaining the problem and the proposed solution. Next step is to test some of the assumptions, such as investigating how to configure the services to setup an environment with a different domain.

**Comment by andy7i** [comment]
Consider making the saturn dev environment as a FiaB. The FiaB can be updated with the latest changes for the other services at whatever interval that Saturn desires. Also, the FiaB's DB can be wiped and seeded or persisted as needed. Another benefit is that the env isn't at the mercy of firecloud dev.

**Comment by katevoss** [comment]
Tech doc (in progress): https://docs.google.com/document/d/1cdfXzyV6JjF129_Niq3n59zIfCORwzeEMHIv3tde3_g/edit?usp=sharing

**Comment by katevoss** [comment]
The tech doc is now fully drafted, moving it over to @zarsky-broad to take a look over. Then it will pass onward to other folks.

**Comment by katevoss** [comment]
I'm going to declare this Spike complete. Many options have discussed and conversations are on-going. It has served the purpose of starting the discussion but it will take more time before we can make any changes.
